helpers/GenerateResourcesAndImage.ps1

```diff
@@ -89,6 +89,9 @@ Function GenerateResourcesAndImage {
         .PARAMETER AzureTenantId
             Tenant needs to be provided for optional authentication via service principal. Example: "11111111-1111-1111-1111-111111111111"
 
+        .PARAMETER RestrictToAgentIpAddress
+            If set, access to the VM used by packer to generate the image is restricted to the public IP address this script is run from
```

Comment:
I think we should add a notice here that the parameter can't be used in combination with `virtual_network_name`, which also exists in our template
![image](https://user-images.githubusercontent.com/48208649/115371262-7d296100-a1d2-11eb-9a83-f8c1cd3fe624.png)

Reply:
Thanks, good point. I will pre-check the wording with my colleagues here and then push a suggestion.

Reply:
Done; let me know if the updated text suits!